src/error.rs

```diff
@@ -114,7 +114,9 @@ impl ApiError {
                     }
                     WeaveError::UnknownError
                 }
-                _ => WeaveError::UnknownError,
+                ValidationErrorKind::FromValidationErrors(ref _err, ref _location) => {
```

Comment:
This seems potentially too broad (all FromValidationErrors are InvalidWbo)? In the python it's only emitted error["location"] = "body" and name in ("bso", "bsos") covered above